.github/workflows/nodejs.yml

```diff
@@ -0,0 +1,25 @@
+name: Node.js CI
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+
+    strategy:
+      matrix:
+        node-version: [10.x]
```

Comment:
Maybe we could try a node version 8 instead than 10? Since this is opensource, we should try to keep as low as possible and Node 8 is still under LTS.